setup.cfg

```diff
@@ -102,6 +102,52 @@ docs =
     matplotlib>=3.3,!=3.4.0,!=3.5.2
     sphinx-changelog>=1.2.0
     Jinja2>=3.0
+dev =
```

Comment:
I recently learned that you can reference other extras here like this: https://github.com/sunpy/sunpy/blob/main/setup.cfg#L126

Reply:
Nice!

Reply:
This is a known issue with the current schema for pyproject.toml files. The packaging forum at PyCon discussed this very problem and decided a PEP would be proposed to enable this in pyproject.toml. I don't know if it has been officially proposed yet or not; however, a group did volunteer to create one.

Reply:
👎 to this change. Just one more list we have to track when we update dependencies, and people are probably going to miss it. And then we have inconsistent requirements.